rpc-integration-test/run.sh

```
@@ -2,50 +1,84 @@
-bitcoind -regtest $BLOCKFILTERARG $FALLBACKFEEARG \
-    -datadir=${TESTDIR}/2 \
-    -connect=127.0.0.1:12348 \
-    -rpcport=12349 \
-    -server=1 \
-    -txindex=1 \
-    -printtoconsole=0 &
-PID2=$!
+# Wait for RPC to be ready
+wait_for_rpc() {
+  local rpcport="$1"
+  for i in {1..60}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)